books/RayTracingInOneWeekend.html

```diff
@@ -171,11 +171,19 @@
     [Listing [first-img]: First image output]
 </div>
 
-If it doesn’t, then you probably just have some newlines or something similar that is confusing the
-image reader.
-
-If you want to produce more image types than PPM, I am a fan of `stb_image.h`, a header-only image
-library available on GitHub at https://github.com/nothings/stb.
+If it doesn’t, then you may just have some line-ending differences or something similar that is
```

Comment:
There is a grammatical mistake here.

```
It should start something like this:
...
If it doesn't, then you may just have some line-ending differences
```

Should look like this -> you may have line-ending differences

This is incorrect.

If your ppm doesn't look like this, the problem is that you formatted the ppm wrong.

If your ppm _does_ look like this, the problem _may_ be line-endings or the like

```
It should start something like this:
...
If the output doesn't look like this you probably have a mistake in how you are printing out.

If the output does like look the above but you're _not_ getting an image, then you may just have some line-ending differences... "
```

Reply:
Changed to
> If your PPM file doesn't look like this, then double-check our formatting code. If it _does_ look like this, then you may have line-ending differences or something similar that is confusing the image reader.